src/packages/auth/adapters/DefaultAuthAdapter.ts

```diff
@@ -0,0 +1,15 @@
+import { getEnv } from '../../../env.js';
+import type { AuthAdapter, ValidateResponse } from "../types.js";
+
+export const DefaultAuthAdapter: AuthAdapter = {
+  logout: async (): Promise<boolean> => true,
```

Comment:
Any way to make this remove the `user` cookie? Then we can potentially make the adapters also in charge of clearing out the cookies that pertain to the interfaces they interact with.

Reply:
n/m. the user token is all managed in the UI server